requirements.yml

```diff
@@ -12,6 +12,7 @@ routing:
   port: 4000
   protocol: http
   subdomain: kafkamon
+  websoket: true
```

Comment:
This is brand new ship-it feature - to enable websocket for the service.

Before that the names of services that require websocket were hardcoded in deployer's code (kafkamon is the only service that requires `websocket` for production)

Reply:
Should that be `websocket` in the requirements.yml?

Reply:
now I wonder how it works...

Reply:
Well, anyway, Thank you, @radenska =)

Indeed, [previous deployment](https://ship-it.internetbrands.com/kubernetes/namespaces/staging/services/kafkamon/deployments/517927) had a warning:
```shell
warning: `routing` > `websoket` is unknown
  --> :15:3
   |
13 |   protocol: http
14 |   subdomain: kafkamon
15 >   websoket: true
   |   ^^^^^^^^^^^^^^ is unknown
```

now it's gone =)